### ISSUE TITLE
Beta: Font refuse to load

### ISSUE THREAD

**Issue by seifsg** [freeCodeCamp/freeCodeCamp#15778]
#### Issue Description
Possibly occuring in all pages.
Opening the developer tool to find these error messages in the console:

```
(index):1 Refused to load the font 'data:font/woff;base64,d09GRgABAAAAAGVUABEAAAAAxuQAAQABAAAAAAAAAAAAAAAAAAAAAAAAAABHREVGAAABgAAAAC4AAAA0ArgC7UdQT1MAAAGwAAAQ6AAALgxKsqRTR1NVQgAAEpgAAAH3AAAELqI5y+RPUy8yAAAUkAAAAE8AAABgaGyBu2NtYXAAABTgAAABlAAAAkQkRATXY3Z0IAAAFnQAAABeAAAAugDsQf1mcGdtAAAW1AAABZcAAAvNb3/BHGdhc3AAABxsAAAACAAAAAgAAAAQZ2x5ZgAAHHQAAEApAAB3CtbiupxoZWFkAABcoAAAADYAAAA2BkubWWhoZWEAAFzYAAAAIAAAACQHFARfaG10eAAAXPgAAAI6AAAEEk4TN4Nsb2NhAABfNAAAAhIAAAISiLhpam1heHAAAGFIAAAAIAAAACACigzgbmFtZQAAYWgAAACUAAABHhQGLdJwb3N0AABh/AAAAq4AAASRk5y6n3ByZ...QxUajCCFt4p9HP4fzdSWs2XhWl5HvJazrIrFUyB0l5dpqcW10lV2wukjMLuAvyMHNiYpgPsrCVXZDKrkpll6UWkh7kABVAFVCDe7UFmxagDegA+hLHRPbqtMo7ZHCpKdT6tPGXybzo0+RXBLoPZt1tELcXxCmAAyZwYTJvdDFZKnDER44X2451rDqCyunIsRWvLSx6wnWqwPj/uX5/KuEy6DL0z6A/Fn79VihxMFJsrlAFy4DpZOcvNlMeNp+BRDLj0r+XFdRxdSNSNxiI/AL3ojKdAAB4AWPw3sFwIihiIyNjX+QGxp0cDBwMyQUbGdictkUwWDAwsDJogTgOPN4c9iz6bMos4iysHFChUDZXJnMWTSZZJrAQt9M+YQYBBh4GTgY2kEZOoJiA0z4GBxiEiDEzuGxUYewIjNjg0BGxkTnFZaMaiLeLo4GBkcWhIzkkAqQkEggceHw5HFkM2VRZJFlYebR2MP5v3cDSu5GJwWUDW9xG1hQXAFAmKZU=' because it violates the following Content Security Policy directive: "font-src *.cloudflare.com https://*.cloudflare.com *.bootstrapcdn.com *.googleapis.com *.gstatic.com https://*.bootstrapcdn.com https://*.optimizely.com 'self'".

(index):1 Refused to load the font 'data:font/woff;base64,d09GRgABAAAAAGz8ABEAAAAA09gAAQABAAAAAAAAAAAAAAAAAAAAAAAAAABHREVGAAABgAAAAC8AAAA0AsQC9UdQT1MAAAGwAAATuAAANLwBEyF1R1NVQgAAFWgAAAIWAAAEZqfk0PVPUy8yAAAXgAAAAFAAAABgaNCCw2NtYXAAABfQAAABkwAAAkQk8AV7Y3Z0IAAAGWQAAABiAAAAugGiQq9mcGdtAAAZyAAABZcAAAvNb3/BHGdhc3AAAB9gAAAACAAAAAgAAAAQZ2x5ZgAAH2gAAESvAAB8yu28l3FoZWFkAABkGAAAADYAAAA2BmibVWhoZWEAAGRQAAAAIAAAACQHMQRzaG10eAAAZHAAAAJDAAAEImBmMbxsb2NhAABmtAAAAhoAAAIaflxdR21heHAAAGjQAAAAIAAAACACjgzgbmFtZQAAaPAAAACdAAABKBQEL8lwb3N0AABpkAAAAsMAAAS9pi3QFXByZ...w76a3jVVUpJzXkBsRtNQoHWTV2mt2UusrulbnIrkvAXNBDFtTVIB8Uoau4pSruq4q7qq2dHpQADUAT0IJ5ra0yPUAfMACMFY6pOtegV/9D7UtTZx72tTeXI4JdcUXh7Pb67D7I/S05AwjAAiYsNie6WOwc4MiYCORSEx+ZExuCvQpiNSRmAdL8wDs2AslUOgp8HfnSYyfCYjrE7w8QDucyS0aXjH0zGk7FX991RgON6L7Qma6pQ+SzA0Qw1x9+HgNFtrBk+F9RsmDpTShvNJL4BDWtP8IAeAFj8N7BcCIoYiMjY1/kBsadHAwcDMkFGxnYnLZFMFgwMLAyaIE4DjzeHPYs+mzKLOIsrBxQoVA2VyZzFk0mWSawELfTPmEGAQYeBk4GNpBGTqCYgNM+BgcYhIgxM7hsVGHsCIzY4NARsZE5xWWjGoi3i6OBgZHFoSM5JAKkJBIIHHh8ORxZDNlUWSRZWHm0djD+b93A0ruRicFlA1vcRtYUFwBQJimV' because it violates the following Content Security Policy directive: "font-src *.cloudflare.com https://*.cloudflare.com *.bootstrapcdn.com *.googleapis.com *.gstatic.com https://*.bootstrapcdn.com https://*.optimizely.com 'self'".

(index):1 Refused to load the font 'data:font/woff;base64,d09GRgABAAAAAGvEABEAAAAA0agAAQABAAAAAAAAAAAAAAAAAAAAAAAAAABHREVGAAABgAAAAC8AAAA0AsQC9UdQT1MAAAGwAAATnQAANLwGhihmR1NVQgAAFVAAAAIWAAAEZqfk0PVPUy8yAAAXaAAAAE4AAABgaTSDymNtYXAAABe4AAABkwAAAkQk8AV7Y3Z0IAAAGUwAAABiAAAAugJwQ3lmcGdtAAAZsAAABZcAAAvNb3/BHGdhc3AAAB9IAAAACAAAAAgAAAAQZ2x5ZgAAH1AAAEOIAAB6jmYk+R5oZWFkAABi2AAAADYAAAA2BoebUWhoZWEAAGMQAAAAIAAAACQHUgSVaG10eAAAYzAAAAJGAAAEImg2Kylsb2NhAABleAAAAhoAAAIaOTEY3W1heHAAAGeUAAAAIAAAACACwAz0bmFtZQAAZ7QAAACiAAABNBUeMO1wb3N0AABoWAAAAsMAAAS9pi3QFXByZ...w76a3jVVUpJzXkBsRtNQoHWTV2mt2UusrulbnIrkvAXNBDFtTVIB8Uoau4pSruq4q7qq2dHpQADUAT0IJ5ra0yPUAfMACMFY6pOtegV/9D7UtTZx72tTeXI4JdcUXh7Pb67D7I/S05AwjAAiYsNie6WOwc4MiYCORSEx+ZExuCvQpiNSRmAdL8wDs2AslUOgp8HfnSYyfCYjrE7w8QDucyS0aXjH0zGk7FX991RgON6L7Qma6pQ+SzA0Qw1x9+HgNFtrBk+F9RsmDpTShvNJL4BDWtP8IAeAFj8N7BcCIoYiMjY1/kBsadHAwcDMkFGxnYnLZFMFgwMLAyaIE4DjzeHPYs+mzKLOIsrBxQoVA2VyZzFk0mWSawELfTPmEGAQYeBk4GNpBGTqCYgNM+BgcYhIgxM7hsVGHsCIzY4NARsZE5xWWjGoi3i6OBgZHFoSM5JAKkJBIIHHh8ORxZDNlUWSRZWHm0djD+b93A0ruRicFlA1vcRtYUFwBQJimV' because it violates the following Content Security Policy directive: "font-src *.cloudflare.com https://*.cloudflare.com *.bootstrapcdn.com *.googleapis.com *.gstatic.com https://*.bootstrapcdn.com https://*.optimizely.com 'self'".

(index):1 Refused to load the font 'data:font/woff;base64,d09GRgABAAAAAG1gABEAAAAA1pAAAQABAAAAAAAAAAAAAAAAAAAAAAAAAABHREVGAAABgAAAAC8AAAA0AsQC9UdQT1MAAAGwAAATbgAANLZecgbjR1NVQgAAFSAAAAIWAAAEZqfk0PVPUy8yAAAXOAAAAFAAAABgaZiEs2NtYXAAABeIAAABkwAAAkQk8AV7Y3Z0IAAAGRwAAABkAAAAugNSRFZmcGdtAAAZgAAABZcAAAvNb3/BHGdhc3AAAB8YAAAACAAAAAgAAAAQZ2x5ZgAAHyAAAEVdAAB/iLdLXbRoZWFkAABkgAAAADYAAAA2BqubS2hoZWEAAGS4AAAAIAAAACQHdQS4aG10eAAAZNgAAAJBAAAEInDeJANsb2NhAABnHAAAAhoAAAIa+cTXSm1heHAAAGk4AAAAIAAAACACwAzxbmFtZQAAaVgAAACbAAABKBPHL31wb3N0AABp9AAAAsMAAAS9pi3QFXByZ...mt41VVKSc15AbEbTUKB1k1dprdlLrK7pW5yK5LwFzQQxbU1SAfFKGruKUq7quKu6qtnR6UAA1AE9CCea2tMj1AHzAAjBWOqTrXoFf/Q+1LU2ce9rU3lyOCXXFF4ez2+uw+yP0tOQMIwAImLDYnuljsHODImAjkUhMfmRMbgr0KYjUkZgHS/MA7NgLJVDoKfB350mMnwmI6xO8PEA7nMktGl4x9MxpOxV/fdUYDjei+0JmuqUPkswNEMNcffh4DRbawZPhfUbJg6U0obzSS+AQ1rT/CAHgBY/DewXAiKGIjI2Nf5AbGnRwMHAzJBRsZ2Jy2RTBYMDCwMmiBOA483hz2LPpsyiziLKwcUKFQNlcmcxZNJlkmsBC30z5hBgEGHgZOBjaQRk6gmIDTPgYHGISIMTO4bFRh7AiM2ODQEbGROcVloxqIt4ujgYGRxaEjOSQCpCQSCBx4fDkcWQzZVFkkWVh5tHYw/m/dwNK7kYnBZQNb3EbWFBcAUCYplQ==' because it violates the following Content Security Policy directive: "font-src *.cloudflare.com https://*.cloudflare.com *.bootstrapcdn.com *.googleapis.com *.gstatic.com https://*.bootstrapcdn.com https://*.optimizely.com 'self'".

(index):1 Refused to load the font 'data:font/woff;base64,d09GRgABAAAAAJd6ABIAAAABdegAAAAAAACWIAAAAVoAAAKDAAAAAAAAAABHUE9TAACDdAAAEl0AAGmo7AqzQkdTVUIAAJXUAAAASgAAAFjau94BTFRTSAAAgzwAAAA4AAABSbMu+GZPUy8yAAAB7AAAAFoAAABgq5xp5lZETVgAAH2EAAADdwAABeBvDHaKY21hcAAAAkgAAAE7AAADVCy12qZjdnQgAACBDAAAAD4AAAA+Aq0JQWZwZ20AAIFMAAABAgAAAXMGWZw3Z2FzcAAAgPwAAAAQAAAAEAATAAlnbHlmAAANwAAAb8MAAOvMFZp2kmhlYWQAAAG0AAAANgAAADb6oXEzaGhlYQAAA4QAAAAhAAAAJAdwBB9obXR4AAADqAAAAr8AAAUUlCcxVmxvY2EAAAs0AAACjAAAAoxSy42GbWF4cAAAAZQAAAAgAAAAIANdBEFuYW1lAAAGaAAAAXwAAANEtvYOJHBvc...hAORYwzQHEUmCaiYGNgYfREUj7MDoBSX+gKCODJwCKqQdxAAB42n2S3WvCMBTF3/tXXPK0wUyqgzGkrYyBL8Mx0LHn2NxqMB9dkrb636/xa4pskIdAzj3n/C7JJlutoEXnpTU5GdKUAJrSCmlWOflcTAfPZFIkmcbABQ/8WlpkjZHfDUoBUuSktJrqXWVN8FRzt/FSW+OtobWzW6n5wNiWD7ytAvWo5dIqQbtqI8OIVm/NiLAia9EI68BwjTmZ7abRi0DjVE7WIdRjxrquO4f0gSxOCfSlk3XoixVZwG0AxWN/NKRYrKWH/nD4wmUcg8pZDUdvCtNGKZCmsk7zaAB8aZsAIY5F+Tj5I3h/Y5ec7IaTJRmLfYqMXXVUskTj8V+wVmJ31E1Uv97h02MaYUtb75xcrcMN6uvpBe7Ke5j1zWB+rPYAo3SYUnjpYfcSDw49uhYFPVe8dHZcYGS7Cfk4MMJ7zwjznvGw3PMA2Oo6+tf/wpSdPlSR/ADe7uPx' because it violates the following Content Security Policy directive: "font-src *.cloudflare.com https://*.cloudflare.com *.bootstrapcdn.com *.googleapis.com *.gstatic.com https://*.bootstrapcdn.com https://*.optimizely.com 'self'".

(index):1 Refused to load the font 'data:font/woff;base64,d09GRgABAAAAAKD5ABIAAAABjrQAAAAAAACfoAAAAVkAAAKCAAAAAAAAAABHUE9TAACLrAAAE6UAAGmo3w+nD0dTVUIAAJ9UAAAASgAAAFjau94BTFRTSAAAi3wAAAAvAAABSW5js1FPUy8yAAAB7AAAAFoAAABgq5xp3FZETVgAAIXAAAADdwAABeBvDHaKY21hcAAAAkgAAAE7AAADVCy12qZjdnQgAACJSAAAADgAAAA4BiEDumZwZ20AAImAAAABAgAAAXMGWZw3Z2FzcAAAiTgAAAAQAAAAEAAaAAlnbHlmAAANvAAAeAEAAQTc9heJHWhlYWQAAAG0AAAANgAAADb6hXFDaGhlYQAAA4QAAAAhAAAAJAdUBBdobXR4AAADqAAAArsAAAUUhLg2g2xvY2EAAAswAAACjAAAAowhImIQbWF4cAAAAZQAAAAgAAAAIANdBGZuYW1lAAAGZAAAAXwAAAM9wTILRnBvc...A5FjDNAcRSYJqJgY2Bh9ERSPswOgFJf6AoI4MnAIqpB3EAAHjafZLda8IwFMXf+1dc8rTBTKqDMaStjIEvwzHQsefY3GowH12Stvrfr/FrimyQh0DOPef8LskmW62gReelNTkZ0pQAmtIKaVY5+VxMB89kUiSZxsAFD/xaWmSNkd8NSgFS5KS0mupdZU3wVHO38VJb462htbNbqfnA2JYPvK0CdbhqFHe0qzYyjGj11owIK7IWjbAODNeYk9luGq0INE7lZB1CPWas67pzRp/H4pRAXzpZh75XkQXcBlA81kdDisVaeugPhy9cxjGonNVw9KYwbZQCaSrrNI8GwJe2CRDiWJSPkz+C9zd2icluMFmSsdinyNhVRyVLNB7/BWsldkfdRPXbHT49phG2tPXOydU63KC+nl7grryHWd8M5sdqDzBKhymFlx52L/Hg0KNrUdBzxUtnxwVGtpuQjwMjvPeMMO8ZD8s9D4CtrqN//S9M2ek/FckP9vfjlA==' because it violates the following Content Security Policy directive: "font-src *.cloudflare.com https://*.cloudflare.com *.bootstrapcdn.com *.googleapis.com *.gstatic.com https://*.bootstrapcdn.com https://*.optimizely.com 'self'".

(index):1 Refused to load the font 'data:font/woff;base64,d09GRgABAAAAAIt0ABEAAAABQDwAAQABAAAAAAAAAAAAAAAAAAAAAAAAAABHUE9TAAABgAAACLoAABJKv3bHbEdTVUIAAAo8AAAAUwAAAHCOOI56T1MvMgAACpAAAABcAAAAYNmuqmljbWFwAAAK7AAAAIEAAAC0jBiQAGN2dCAAAAtwAAAAKwAAAC4G9xijZnBnbQAAC5wAAAOBAAAG5XJackBnYXNwAAAPIAAAAAwAAAAMABIAGGdseWYAAA8sAABYNwAAw2DADxfjaGVhZAAAZ2QAAAA2AAAANvyc8iNoaGVhAABnnAAAAB8AAAAkD7YHdWhtdHgAAGe8AAACCwAAA25yNU6fa2VybgAAacgAAB0cAABXGD87P7Jsb2NhAACG5AAAAb4AAAG+rN9/im1heHAAAIikAAAAIAAAACAB1wf5bmFtZQAAiMQAAADJAAABeBwnOJpwb3N0AACJkAAAAZYAAAJBVuQNNnByZ...CRYydOnTl34dKVazdu3bn34NGTZy9evXn34dOXbz9+/fkXEhYRFROXkCwRBA/LYUABAAD3Td3+ZW3bnTp2LrFt2+bEPMdOztkNF+T65bd2KTb8ESNSumJ54aIIy35KdORYtFT/9Fp3KEOJUyfO5Cg3bFCFm26Jc9uoO4aMmDRm3IRNd82YMq3SPQfizZs1575tu/576IFHnnjsqSzPvPDcS6+88dpb72x576MPPvnisybZvvnqux927GmxoEq1RWuW1KjVoFGfOvX6/dWhU1u4ZF+X7nA5XAlXw7VwPdw4H4tfqI8Pe2lepoGBowGUdmFxLS3K50ovSixL1UtOLE7lTclMLUotziwG87gSk0tLIBL8yZlFyaW5aTmpFWA+Z1FmXjpEUUlmTgpEEQDcm4MFAAC5CAAIAGMgsAEjRCCwAyNwsBRFICCwKGBmIIpVWLACJWGwAUVjI2KwAiNEswkKAwIrswsQAwIrsxEWAwIrWbIEKAZFUkSzCxAEAisA' because it violates the following Content Security Policy directive: "font-src *.cloudflare.com https://*.cloudflare.com *.bootstrapcdn.com *.googleapis.com *.gstatic.com https://*.bootstrapcdn.com https://*.optimizely.com 'self'".

(index):1 Refused to load the font 'data:font/woff;base64,d09GRgABAAAAAI3gABIAAAABRWQAAQABAAAAAAAAAAAAAAAAAAAAAAAAAABHUE9TAAABlAAACMoAABJKyujMg0dTVUIAAApgAAAAUwAAAHCOOI56T1MvMgAACrQAAABcAAAAYNk4qA5jbWFwAAALEAAAAIEAAAC0jBiQAGN2dCAAAAuUAAAAGgAAABoA7AimZnBnbQAAC7AAAAECAAABcwZZnDdnYXNwAAAMtAAAAAwAAAAMACIAGGdseWYAAAzAAABSygAAubBheDH9aGRteAAAX4wAAAoHAAAUKE4Jg9doZWFkAABplAAAADYAAAA2/E/x4WhoZWEAAGnMAAAAHwAAACQPaQc8aG10eAAAaewAAAIMAAADbmB1YDdrZXJuAABr+AAAHTYAAFcYYDRhP2xvY2EAAIkwAAABvgAAAb6LJGLGbWF4cAAAivAAAAAgAAAAIAL2BApuYW1lAACLEAAAANMAAAF4HEI4H3Bvc...VQUSWVVVFVNdXVUFMttdVRVz31NdBQI4010VQzzbXQUiuttdFWO+11CAmz1iSTHbfEC1PMMdNym60LCTfDYxMt9Mlnsy01zVnPfbTCFl998c0a21120Q4ddTJPZ1d1cckVN11z3Q0vdXXHLbft1M0H89131z3dvfbWdD310EsfvfW1Sj8D9DfQIEMMNtQwrww30gijjDHaIauNM9Z4E7zxzhEP7LLbQ888ssdeBxx0zj77nTfVCScdC4nw3imnQyJDokKiQ2JCYkMC4XWb1K4dNaRvj2CwQvCfK4dX+bnSonyu9KLEslS95MTiVN6UzNSi1OLMYjCPKzG5tAQiwZ+cWZRcmpuWk1oB5nMWZealQxSVZOakQBQBAC/zgrd4AdvBwKDNsIuBkYGJgUmbcRcDM4SxH8goYYhn8GIwZVBgYGDg0N7PwMIwnaGGIYHBlUELLMKwH6htK8MUhmKGIAYjqCImVEUgs1mBOtm1gRYpuNZmSrgAALqcEVgAAAA=' because it violates the following Content Security Policy directive: "font-src *.cloudflare.com https://*.cloudflare.com *.bootstrapcdn.com *.googleapis.com *.gstatic.com https://*.bootstrapcdn.com https://*.optimizely.com 'self'".

(index):1 Refused to load the font 'data:font/woff;base64,d09GRgABAAAAAI7kABEAAAABPbQAAQABAAAAAAAAAAAAAAAAAAAAAAAAAABHUE9TAAABgAAACOkAABMM+T2iU0dTVUIAAApsAAAAUwAAAHCOOI56T1MvMgAACsAAAABcAAAAYNmuqeZjbWFwAAALHAAAAIEAAAC0jBiQAGN2dCAAAAugAAAAKwAAAC4G8BiZZnBnbQAAC8wAAAOBAAAG5XJackBnYXNwAAAPUAAAAAwAAAAMABEAGGdseWYAAA9cAABcdgAAxExusxknaGVhZAAAa9QAAAA2AAAANvuq8Z1oaGVhAABsDAAAACIAAAAkD8gOJGhtdHgAAGwwAAACLgAAA24+wkYha2VybgAAbmAAABvuAABS7DoyQgNsb2NhAACKUAAAAb4AAAG+pFt11G1heHAAAIwQAAAAIAAAACAB3gf5bmFtZQAAjDAAAADIAAABcBvjNsxwb3N0AACM+AAAAZ0AAAJBVtsNMnByZ...JKKKmU0sooq5zyKqioksqqqKqa6mqoqZba6qirnvoaaKiRxppoqpnmWmipldbaaKud9jroqJPOuugaFGKtSSY7bokXpphjpuU2WxcUaobHJlrok89mW2qas577aIUtvvrimzW2u+yiHbrpbp4erurpkituuua6G17q5Y5bbtuptw/mu++ue/p47a3p+umrv4EGGGSVwWIMEStOgnhDDfPKcCONMMoYox2y2jhjjTfBG+8c8cAuuz30zCN77HXAQefss995U51w0rGgMO+dcjooPCgiKDIoKig6KBDaqGWDBhEJf5aXaWDgaAClXVhcS4vyudKLEstS9ZITi1N5UzJTi1KLM4vBPK7E5NISiAR/cmZRcmluWk5qBZjPWZSZlw5RVJKZkwJRBAA2vYT3AAAAuQgACABjILABI0QgsAMjcLAURSAgsChgZiCKVViwAiVhsAFFYyNisAIjRLMJCgMCK7MLEAMCK7MRFgMCK1myBCgGRVJEswsQBAIrAA==' because it violates the following Content Security Policy directive: "font-src *.cloudflare.com https://*.cloudflare.com *.bootstrapcdn.com *.googleapis.com *.gstatic.com https://*.bootstrapcdn.com https://*.optimizely.com 'self'".

(index):1 Refused to load the font 'data:font/woff;base64,d09GRgABAAAAAEzMABMAAAAAezAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAABGRlRNAAABqAAAABwAAAAcbHuVbkdERUYAAAHEAAAAHAAAAB4AJwCWR1BPUwAAAeAAAAXZAAAJpGVqwf1HU1VCAAAHvAAAACwAAAAwuP+4/k9TLzIAAAfoAAAAXAAAAGDY59IiY21hcAAACEQAAAFPAAACCqtY6vFjdnQgAAAJlAAAADcAAABIJlMGxGZwZ20AAAnMAAAFqAAAC5fbFNvwZ2FzcAAAD3QAAAAIAAAACAAAABBnbHlmAAAPfAAAM+AAAE6g3+9Ct2hlYWQAAENcAAAAMwAAADYIhrfwaGhlYQAAQ5AAAAAhAAAAJA5MBZ5obXR4AABDtAAAAboAAAJA90wzR2xvY2EAAEVwAAABEAAAASIQ0fv8bWF4cAAARoAAAAAgAAAAIAG7Ac1uYW1lAABGoAAABEkAAAx4Zctph3Bvc...c8CSSSRDIppJJGOhlkkkU2OeSSRz4FFFJEMSWUUkY5FWjoOHFRiZsqqqmhljrqaaCRJpppwRP4rg0v7fjooJMuuumhlz76GWCQIYYZYZQxxplgkimmmWGWOeZZYJElCeaAQy4445hLsXDEKZ9c8cYH72KVEAkVm4RJuESIXSLFIVESLTESyw23PPDIC3fc88q1xPHEs8RLAieSaFnb3N9Z163+rQ1N07xGdBnRoym9vzoDe6WudCpdykqlW1mlrFbWKGuVHkNd3dV12+rGmn93ZXlpb90YOX2Gbp+53b+7/RfcvrYfygRb7gAAAHjaY/DewXAiKGIjI2Nf5AbGnRwMHAzJBRsZ2Jw2M0gzMWiBWFsVGfk4mDggbCUGCTYwm8NpN0cD8wEGJgZOII/LaTcDA4MDhMfM4LJRhbEjMGKDQ0fERuYUl41qIN4ujgYGRhaHjuSQCJCSSCAAGifAwcSjtYPxf+sGlt6NTECdrCkuALy2JpcAAVT4eScAAA==' because it violates the following Content Security Policy directive: "font-src *.cloudflare.com https://*.cloudflare.com *.bootstrapcdn.com *.googleapis.com *.gstatic.com https://*.bootstrapcdn.com https://*.optimizely.com 'self'".

(index):1 Refused to load the font 'data:font/woff;base64,d09GRgABAAAAAIw4ABEAAAABQcAAAQABAAAAAAAAAAAAAAAAAAAAAAAAAABHUE9TAAABgAAACU8AABMOD1XXMEdTVUIAAArQAAAAUwAAAHCOOI56T1MvMgAACyQAAABbAAAAYNrnrZBjbWFwAAALgAAAAIEAAAC0jBiQAGN2dCAAAAwEAAAAKgAAAC4HyBmgZnBnbQAADDAAAAOBAAAG5XJackBnYXNwAAAPtAAAAAwAAAAMAA0AGGdseWYAAA/AAABViQAAvQzAr6AGaGVhZAAAZUwAAAA2AAAANvzd8kdoaGVhAABlhAAAAB8AAAAkD/YHpGhtdHgAAGWkAAACFAAAA26BW0HVa2VybgAAZ7gAAB/3AABeRKoEucNsb2NhAACHsAAAAb4AAAG+1eip6m1heHAAAIlwAAAAIAAAACAByAf2bmFtZQAAiZAAAADEAAABYBoFNRJwb3N0AACKVAAAAZYAAAJBVvgNXnByZ...KKKqa4EkoqpbQyyiqnvAoqqqSyKqqqproaaqqltjrqqqe+BhpqpLEmmmqmuRZaaqW1Ntpqp70OOuqksy66BhE2m2Gms1Z5Z5ZF5ltrpy1BpHlemm65H35aaLU5Lnvru3V2+e2XPzbZ66br9ummuyV6uK2nG26574677nmvl0ceeGi/3r5Z6qnHnujjo8/m6qev/gYaYJANBhtqiGGGG2mEUUb7YIxxxhpvoglO2GiySaaY6pMvTnnmgIOee+OFQw475rgrjjjqqtnOOe9MEOWrCy4G0UFMEBvEBfFB+N9Y/EJ9fNhL8zINDBwNoLQLi2tpUT5XelFiWapecmJxKm9KZmpRanFmMZjHlZhcWgKR4E/OLEouzU3LSa0A8zmLMvPSIYpKMnNSIIoAYXqDQQAAuQgACABjILABI0QgsAMjcLAURSAgsChgZiCKVViwAiVhsAFFYyNisAIjRLMJCgMCK7MLEAMCK7MRFgMCK1myBCgGRVJEswsQBAIrAA==' because it violates the following Content Security Policy directive: "font-src *.cloudflare.com https://*.cloudflare.com *.bootstrapcdn.com *.googleapis.com *.gstatic.com https://*.bootstrapcdn.com https://*.optimizely.com 'self'".

```

#### Browser Information

* Browser Name, Version: Chrome only
* Operating System: Windows 10
* Mobile, Desktop, or Tablet: Desktop 

#### Screenshot

![image](https://user-images.githubusercontent.com/6075316/29489601-3835a554-8525-11e7-9f5c-0f9f16aed236.png)

**Comment by vivek-agrawal27** [comment]
When I tried to reproduce this in opera (which is based on chromium) I wasn't able to do so. But it's only happening with chrome as mentioned by you. The thing here is that the website looks exactly same on both the browsers. 
We'll have to look into this deeply. 
@QuincyLarson Thoughts?

**Comment by DusanSacha** [comment]
Could you post the exact link where did you reproduce the error?
I have the same configuration as mentioned in the first post, I don't see any console errors though. (Temporary problem possibly?)

**Comment by zk433** [comment]
I see the same issue on every beta page. This, for example, is from this link:

[](http://beta.freecodecamp.com/en/challenges/managing-packages-with-npm/introduction-to-the-managing-packages-with-npm-challenges)

![image](https://user-images.githubusercontent.com/15965102/29564678-bf303e72-873a-11e7-86f7-d186a9ee3536.png)

**Comment by zk433** [comment]
I see the same issue on every beta page. This, for example, is from [this ](http://beta.freecodecamp.com/en/challenges/managing-packages-with-npm/introduction-to-the-managing-packages-with-npm-challenges) link.

![image](https://user-images.githubusercontent.com/15965102/29564705-dd13b3d8-873a-11e7-9159-bae5bcdd87d6.png)

**Comment by QuincyLarson** [comment]
This looks like an issue with the Content Security Policy. CC @BerkeleyTrue

**Comment by BerkeleyTrue** [comment]
I'm not aware of any fonts that we use that are sourced as data blobs. Our fonts are set up as .tiff files.

![basic_html_and_html5__say_hello_to_html_elements___freecodecamp_](https://user-images.githubusercontent.com/6775919/29597626-c26fdf1e-8778-11e7-9f00-31d38b1e7cda.png)

These are the totality of all our fonts.

I suspect some errant third party extension

**Comment by zk433** [comment]
Just to update you all, I still get the errors, in FCC, in Beta and while running FCC locally as well. It produces 33 errors every time.

**Comment by QuincyLarson** [comment]
@zk433 are you getting the same errors reported by @seifsg? Can you look in the staging branch and see whether any of these fonts are referred to anywhere in there? If not, it may be a dependency that's trying to load them.

**Comment by zk433** [comment]
@QuincyLarson Yes, it's exactly the same issue every single time. And always repeats 33 times.

I looked through the codebase and I couldn't find anything there. But based on some googling, it seems to be related to the Content Security Policy set up: 'font-src': "data:" should be added to it to allow for fonts to be loaded from data: URIs. I did find the content-security-policy-builder dependency, but that's as far as I got since I have no idea what it is that I'm looking for - this is way over my head, sorry!

**Comment by QuincyLarson** [comment]
@zk433 Thanks for doing this research. 

@BerkeleyTrue would using `font-src': "data:"` like @zk433 proposed open us up to any additional threats that you can think of? If so, it sounds like this might solve this minor issue that several campers have independently reported.

**Comment by BerkeleyTrue** [comment]
Yes, it is possible that using font-src data would open us up to some volnurabilities.

Just to reiterate, we don't use any fonts in the form of data blobs. Neither should the page linked above require any fonts other than what we already link to. This is most likely a third part extension issue.

**Comment by QuincyLarson** [comment]
@BerkeleyTrue OK - thanks for the clarification.

@zk433 Can you try to open this in incognito tab with all of your Chrome extensions turned off and see whether you're still able to reproduce this issue?

**Comment by zk433** [comment]
@BerkeleyTrue , @QuincyLarson , yes 👍 ! There were no errors when using the incognito tab so I went through all my extensions and found that it was the Grammarly Chrome extension that was causing the errors! As soon as I disabled it, errors went away.

Well, that turned out to be a much simpler solution than I thought it would be :). But I learned a lot about the Content Security Policy so time well spent :)
